lib/Monocle_Alternate.php

```diff
@@ -54,6 +70,7 @@ public function get_active_by_id($ids, $swLat, $swLng, $neLat, $neLng)
         $params = array();
```

Comment:
I think something went wrong when solving the merge conflicts, `$miniv, $minlevel, $exminiv` are removed from function params.